runtime/parachains/src/paras/mod.rs

```diff
@@ -467,6 +477,12 @@ pub mod pallet {
 		NewHeadNoted(ParaId),
 		/// A para has been queued to execute pending actions. `para_id`
 		ActionQueued(ParaId, SessionIndex),
+		/// The given para either initiated or subscribed to a PVF check for the given validation
+		/// code. `code_hash` `para_id`
+		PvfCheckStarted(ValidationCodeHash, ParaId),
+		/// The PVF pre-checking for the given validation code was concluded with the given outcome
+		/// (either accepted or rejected). `code_hash` `para_id` `accepted`
+		PvfCheckConcluded(ValidationCodeHash, ParaId, bool),
```

Comment:
IMO we could also just split this up into two variants.

Reply:
What's your reasoning here?

Reply:
You directly see the output based on the event name.

Reply:
Ok, don't feel about this strongly. [Here](https://github.com/paritytech/polkadot/compare/9537c19cb666468688f7e4bba3c81877ffd773cf..bec1cc031c7b0f8942ca8c3d006952f1a092b974) is the fix